next/components/Molecules/Carousel/Carousel.tsx

```diff
@@ -9,6 +10,7 @@ interface SectionPromosProps {
   items: { element: ReactNode; key: string | undefined }[]
   shiftIndex?: number
   visibleItemsCount?: number
+  hideScrollbar?: boolean
```

Comment:
We have discussed this a little in the office and we decided to hid the scroll bar for carousel for now. I know it's maybe not the wanted everywhere, but it needs more discussion/research and more systematic approach. For now, there are other indicators that the section is scrollable such as arrows on desktop and partly visible card on mobile screen.

So I would prefer to keep it consistent with no scroll bar everywhere at the moment. But if you have some other instructions, then go for it :D

Reply:
Scrollbar is hidden in desktop mode, with smaller screens it will show scrollbar, looks like component "book news"
Carousel was edited just for this case, but defaults to same look as before
AFIK there are no arrows for this components for now